src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,46 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, K> {
 
+    private T first;
+    private K second;
+
+    private Pair(T first, K second) {
+        this.first = first;
+        this.second = second;
+    }
+
+    public static <T, K> Pair<T, K> of(T first, K second) {
+        return new Pair<T, K>(first, second);
```

Comment:
```suggestion
        return new Pair<>(first, second);
```
we can omit indicating types here because of the [diamond operator](https://www.baeldung.com/java-diamond-operator)